scripts/ci/lib.sh

```diff
@@ -1121,11 +1121,14 @@ post_process_test_results() {
         fi
 
         csv_output="$(mktemp --suffix=.csv)"
-
+        # We need a link to repository. In case it's not part of job spec (e.g., periodic`s)
+        # we will fallback to short commit
+        local base_link="$(echo "$JOB_SPEC" | jq ".refs.base_link | select( . != null )" -r)"
+        local calculated_base_link="https://github.com/stackrox/stackrox/commit/$(make --quiet --no-print-directory shortcommit)"
```

Comment:
Maybe this could be more reusable if we took org and repo from:
```
 "extra_refs": [
    {
      "org": "stackrox",
      "repo": "stackrox",
      "base_ref": "master"
    }
  ],
```

Reply:
I thought about this but this adds extra complexity. Although we already do that in at least 2 places
https://github.com/stackrox/stackrox/blob/ee590c92138ef27d787fbaf2cdab26d006c91763/scripts/ci/lib.sh#L1191-L1200
https://github.com/stackrox/stackrox/blob/ee590c92138ef27d787fbaf2cdab26d006c91763/scripts/ci/lib.sh#L859-L860